ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/tracing/flat/FlatTraceGenerator.java

```diff
@@ -124,25 +129,25 @@
               + traceFrame.getPrecompiledGasCost().orElse(Gas.ZERO).toLong();
 
       final String opcodeString = traceFrame.getOpcode();
-      if ("CALL".equals(opcodeString)
-          || "CALLCODE".equals(opcodeString)
-          || "DELEGATECALL".equals(opcodeString)
-          || "STATICCALL".equals(opcodeString)) {
-
-        currentContext =
-            handleCall(
-                transactionTrace,
-                traceFrame,
-                nextTraceFrame,
-                flatTraces,
-                cumulativeGasCost,
-                tracesContexts,
-                opcodeString.toLowerCase(Locale.US));
-
-      } else if ("CALLDATALOAD".equals(opcodeString)) {
-        if (currentContext != null) currentContext = handleCallDataLoad(currentContext, traceFrame);
-      } else if ("RETURN".equals(opcodeString) || "STOP".equals(opcodeString)) {
-        if (currentContext != null) {
+      if (currentContext != null) {
```

Comment:
since this wraps all of the remaining conditions should it be changed to a mini guard-block?

```
if (currentContext == null) {
  continue;
}
```
or even a `break` since there is no way the currentContext will ever get changed?

Reply:
Done . I will use break